src/go/supervisor/pkg/svc/manage.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2021 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package svc
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+func StartService(name string) error {
+	m, err := mgr.Connect()
+	if err != nil {
+		return err
+	}
+	defer m.Disconnect()
+	s, err := m.OpenService(name)
+	if err != nil {
+		return fmt.Errorf("could not access service: %w", err)
+	}
+	defer s.Close()
+	if err = s.Start("is", "manual-started"); err != nil {
+		return fmt.Errorf("could not start service: %w", err)
+	}
+	return nil
+}
+
+func ControlService(name string, c svc.Cmd, to svc.State) error {
+	m, err := mgr.Connect()
+	if err != nil {
+		return err
+	}
+	defer m.Disconnect()
+	s, err := m.OpenService(name)
+	if err != nil {
+		return fmt.Errorf("could not access service: %w", err)
+	}
+	defer s.Close()
+	status, err := s.Control(c)
+	if err != nil {
+		return fmt.Errorf("could not send control=%d: %w", c, err)
+	}
+	timeout := time.Now().Add(10 * time.Second)
+	for status.State != to {
```

Comment:
Ugh, there's nothing in the API so we have to poll for the state don't we :( But this should only apply if the service state is pending (`StartPending`, `StopPending`, `ContinuePending`, `PausePending`)?